dfttopif/parsers/pwscf.py

```diff
@@ -29,6 +29,11 @@ def get_result_functions(self):
 
     def get_name(self): return "PWSCF"
 
+    def _get_key_with_units(self, key):
+        if key not in self.settings:
+            return None
+        return Property(scalars=[self.settings[key]], units=self.settings["{} units".format(key)])
```

Comment:
Is there a reason to wrap in a list here? That would cause issues if the value at self.settings[key] was already a list - doubly-nested lists are not allowed in the scalars field.

Reply:
This is only used when `self.settings[key]` is an atomic value.  `scalars` must be a list (server-side standardization aside), so its correct to wrap it in a list.